nvflare/fuel/f3/cellnet/cell.py

```diff
@@ -371,10 +371,11 @@ def _register_request_cb(self, channel: str, topic: str, cb, *args, **kwargs):
 
         if not callable(cb):
             raise ValueError(f"specified request_cb {type(cb)} is not callable")
+
+        # always register with core_cell since some requests (e.g. broadcast_multi_requests) will directly go
+        # through the core_cell, even if the channel may be a stream channel (e.g. aux channel).
+        self.core_cell.register_request_cb(channel, topic, cb, *args, **kwargs)
```

Comment:
This means the multi-cast doesn't go through streaming cell. Wait for multiple streaming channels are too complicated and I think this is the right decision. We probably should make it clear somewhere on what condition a channel is streamed.

Reply:
That's right. We could enhance this later if needed (which I doubt).